anything-llm/connector.py

```diff
@@ -0,0 +1,22 @@
+""" Copyright start
```

Comment:
please update the certificate to this
"""
Copyright start
MIT License
Copyright (c) 2024 Fortinet Inc
Copyright end
"""